README.md

```diff
@@ -48,7 +48,8 @@ exactly what to execute. Create a file named `.atom-build.json` in your project
         "VARIABLE2": "VALUE2",
         ...
       },
-      "errorMatch": "^regexp$"
+      "errorMatch": "^regexp$",
+      "buildViewTimeout": 1000
```

Comment:
I don't think you'd want to configure this differently for different builds. I think this should be a configuration which is global instead.